GravityWall/Assets/Scripts/Module/PlayTest/GameClearObject.cs

```diff
@@ -7,13 +7,17 @@ namespace Module.PlayTest
 {
     public class GameClearObject : MonoBehaviour
     {
-        [SerializeField] private string SceneName;
+        private RespawnManager respawnManager => RespawnManager.instance;
+        [SerializeField]private GameObject ClearCanvas;
 
         private void OnTriggerEnter(Collider other)
         {
             if (other.tag == "Player")
```

Comment:
```suggestion
            if (other.tag == "Player")
```
タグを使うときはTagクラスの定数を使ってください。